docs/docs/unit-testing.md

```diff
@@ -126,6 +125,22 @@ global.___loader = {
 }
 ```
 
+Finally it's a good idea to mock the gatsby module itself. This may not be
+needed at first, but will make thinmgs a lot easier if you want to test
+components that use `Link` or GraphQL.
+
+```js
```

Comment:
Where do you put the content? Which file?

Reply:
Added